apps/browser/qml/pages/components/PrivacySettingsConfirmDialog.qml

```diff
@@ -0,0 +1,147 @@
+                //% "Confirm"
+                acceptText: qsTrId("sailfish_browser-he-confirm");
+            }
+
+            Label {
+                x: Theme.horizontalPageMargin
+                width: parent.width - 2*Theme.horizontalPageMargin
+
+                //% "The following browser data will be deleted:"
+                text: qsTrId("sailfish_browser-la-following_browser_data_will_be_deleted:");
+                highlighted: true
+                wrapMode: Text.Wrap
+            }
+
+            BrowserDataItem {
+                id: historyItem
+
+                text: {
+                    switch(historyPeriod) {
+                    case 1:
+                        //% "Clear browser history for the last 24 hours"
+                        return qsTrId("sailfish_browser-la-clear_browsing_history_for_24_hours")
+                    case 7:
+                        //% "Clear browser history for the last week"
+                        return qsTrId("sailfish_browser-la-clear_browsing_history_for_last_week")
+                    case 28:
+                        //% "Clear browser history for the last 4 weeks"
+                        return qsTrId("sailfish_browser-la-clear_browsing_history_for_last_four_weeks")
+                    default:
+                        //% "Clear all of the browser history"
+                        return qsTrId("sailfish_browser-la-clear_all_browsing_history")
+                    }
+                }
+            }
+            BrowserDataItem {
+                id: cookieAndSiteDataItem
+
+                text: _siteDataUsage > 0
+                        //% "%1 of cookie and site data"
+                      ? qsTrId("sailfish_browser-la-usage_of_cookie_and_site_data").arg(Format.formatFileSize(_siteDataUsage))
+                        //% "Cookie and site data"
+                      : qsTrId("sailfish_browser-la-cookie_and_site_data")
+            }
+
+            BrowserDataItem {
+                id: passwordsItem
+
+                //% "Saved passwords"
+                text: qsTrId("sailfish_browser-la-saved_passwords");
+            }
+
+            BrowserDataItem {
+                id: cacheItem
+
+                text: _siteDataUsage > 0
+                        //% "%1 of cache"
+                      ? qsTrId("sailfish_browser-la-usage_of_cache").arg(Format.formatFileSize(_cacheUsage))
+                        //% "Cache"
+                      : qsTrId("sailfish_browser-la-cache")
+            }
+
+            BrowserDataItem {
+                id: bookmarksItem
+
+                //% "Bookmarks"
+                text: qsTrId("sailfish_browser-la-bookmarks");
+            }
+
+            BrowserDataItem {
+                id: sitePermissionsItem
+
+                //% "Site permissions"
+                text: qsTrId("sailfish_browser-la-site_permissions");
+            }
+        }
+    }
+
+    onAccepted: {
```

Comment:
Currently when the page is accepted it works nicely, and then moves back to the "Clear private data" page. I'm thinking it might be nicer if it moved back to the "Settings" page instead. What do you think?